### PR TITLE
Fixing splunk test module

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22202

## Description
change test module to a lite query.

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

### CODE REVIEW

File: Integrations/SplunkPy/SplunkPy.py [hunk 1]
```diff
@@ -555,18 +555,11 @@ def splunk_parse_raw_command():
...
-        t = datetime.utcnow() - timedelta(days=3)
+        t = datetime.utcnow() - timedelta(hours=1)
         time = t.strftime(SPLUNK_TIME_FORMAT)
         kwargs_oneshot = {'count': 1, 'earliest_time': time}
         searchquery_oneshot = demisto.params()['fetchQuery']
-        oneshotsearch_results = service.jobs.oneshot(searchquery_oneshot, **kwargs_oneshot)  # type: ignore
-        reader = results.ResultsReader(oneshotsearch_results)
-        for item in reader:
-            if item:
-                demisto.results('ok')
```
Comment (esharf) [inline Integrations/SplunkPy/SplunkPy.py]: if the query will be invalid we get an exceptin.
so we don't need to check results.

File: Integrations/SplunkPy/SplunkPy.py [hunk 2]
```diff
@@ -555,18 +555,11 @@ def splunk_parse_raw_command():
 
 def test_module(service):
     if demisto.params().get('isFetch'):
-        t = datetime.utcnow() - timedelta(days=3)
```
Comment (esharf) [inline Integrations/SplunkPy/SplunkPy.py]: see the comment about the invalid query.
we don't need really results so one hour is enough.

File: Integrations/SplunkPy/SplunkPy.py [hunk 3]
```diff
@@ -602,6 +595,7 @@ def main():
     # The command demisto.command() holds the command sent from the user.
     if demisto.command() == 'test-module':
         test_module(service)
+        demisto.results('ok')
```
Comment (esharf) [inline Integrations/SplunkPy/SplunkPy.py]: Connect is already checked.

File: Integrations/SplunkPy/SplunkPy.py (hunk 1 above)
Comment (IkaDemisto) [reply Integrations/SplunkPy/SplunkPy.py]: But the main point in this test is to validate that the search query input is valid. Don't think that we should remove it

Comment (IkaDemisto) [reply Integrations/SplunkPy/SplunkPy.py]: What will happen if query is empty, it is not mandatory field (for some reason) and has no default value. Try to run it  with empty query and see the error

Comment (esharf) [reply Integrations/SplunkPy/SplunkPy.py]: `isFetch` should be set to `False`

**Comment by kirbles19** [comment]
@esharf Done.

**Pull request merged**